packages/apollos-api/config.yml

```diff
@@ -27,6 +27,9 @@ TWILIO:
   ACCOUNT_SID: ${TWILIO_ACCOUNT_SID}
   AUTH_TOKEN: ${TWILIO_AUTH_TOKEN}
   FROM_NUMBER: ${TWILIO_FROM_NUMBER}
+YOUTUBE:
+  API_KEY: ${YOUTUBE_API_KEY}
+  PLAYLIST_ID: 'UUmFRqL-BDXeaYwMxYDfhu_w'
```

Comment:
Before this is merged I want to move this from using a single playlist to playlists for each campus. I'm still thinking this might be in Rock, so going to leave this open for now.

Reply:
Solution here which we can build off later
https://github.com/Differential/willow-prototype/pull/7/commits/6a23cb12982a4e82ec92efcc4367263784f89f8e